### PR TITLE
Component inbox

### PR DESCRIPTION
**Pull request by saralaffin**
- Related Issue (include '#'): #22 

- Description of changes made: Created Inbox component

- Is the feature complete/bug resolved/etc..: 
Think it still needs some sizing work when integrated into site (I left a comment in CSS file). 

- Screenshot(s):
image

### CODE REVIEW

**Comment by saralaffin** [comment]
Oops! Looks like I accidentally added some work from the onClick NavBar issue I ran into. Thought I was on a different branch... I'll try to change that!

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+django = "*"
+knox = "*"
+django-rest-knox = "*"
```
Comment (ZakkMan) [inline Pipfile:11]: Hey, what's this doing here?

Comment (ZakkMan) [reply Pipfile:11]: Same with the Pipfile.lock

File: api/cotrip_app/migrations/0002_location.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.0.4 on 2020-03-30 18:46
+
+from django.db import migrations, models
+
+
```
Comment (ZakkMan) [inline api/cotrip_app/migrations/0002_location.py:5]: What's this doing here?

Comment (saralaffin) [reply api/cotrip_app/migrations/0002_location.py:5]: Uhh.. I have no idea! I didn't think I did any work on the backend.. will try to exclude those

**Comment by saralaffin** [comment]
I tried deleting those files... does that work?

**Comment by saralaffin** [comment]
Actually my brother suggested closing this one and starting with a fresh branch... so I'll go try that!

**Pull request closed without merging**